Build/Scripts/EnlistmentHelperFunctions.psm1

```diff
@@ -0,0 +1,68 @@
+function Get-GitBranchName() {
+    if ($ENV:GITHUB_REF_NAME) {
+        return $ENV:GITHUB_REF_NAME
+    }
+    return git rev-parse --abbrev-ref HEAD
+}
+
+function Get-BaseFolder() {
+    if ($ENV:GITHUB_WORKSPACE) {
```

Comment:
Let's revisit that approach.

Your other idea about having init script sounds promising.

Reply:
I agree :) Should we keep it as it is for now, and then clean it up once we have another approach in place?

Reply:
This you cannot keep as is, because it's in the main branch already. Otherwise, up to you.